spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/autoconfiguration/XsuaaAutoConfigurationTest.java

```diff
@@ -70,7 +70,7 @@ public void configures_xsuaaRestTemplate() {
 	}
 
 	@Test
-	public void configures_xsuaaMtlsRestTemplate() {
+	public void configures_xsuaaMtlsRestTemplateWithCredType() {
```

Comment:
I think this test doesn't make any sense any longer, because even without credential-type it would work same like the test below `configures_xsuaaMtlsRestTemplate()` as the condition NoClientCertificateCondition evaluates to false. What would make sense instead would be cases where this condition evaluates to true.